loom/script/serialize/lsBinWriter.cpp

```diff
@@ -644,6 +644,7 @@ void BinWriter::writeAssembly(json_t *json)
             lmAssert(jbinary, "Error with linked assembly %s, missing binary section", refname);
             utString  refjson  = (const char *)utBase64::decode64(json_string_value(jbinary)).getData().ptr();
             BinWriter *bwriter = lmNew(NULL) BinWriter;
+            lmAssert(refjson.length() > 0, "Refjson should not be empty! %s", json_string_value(jbinary));
```

Comment:
I had it happen to me, I can't remember the exact circumstances, but I think it's a useful check.